modular_skyrat/master_files/code/modules/client/preferences/_preference.dm

```diff
@@ -95,15 +108,53 @@
 	var/datum/sprite_accessory/default_accessory_type
 	/// Path to the corresponding /datum/preference/toggle to check if part is enabled.
 	var/datum/preference/toggle/type_to_check
+	/// Generates icons from the provided mutant bodypart for use in the selection box.
+	var/generate_icons = FALSE
+	/// A list of the four co-ordinates to crop to. Useful for icons whos main contents are smaller than 32x32. Please keep it square.
```

Comment:
Should also mention what the use-case of it is (to be used with generate_icons I reckon).
```suggestion
	/// A list of the four co-ordinates to crop to. Useful for icons whose main contents are smaller than 32x32. Please keep it square.
```